specs/master/cluster-init/scripts/030_install_scalelib.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+HF_TOP=$( jetpack config symphony.hostfactory.top )
+if [ -z "${HF_TOP}" ]; then
+   # In Symphony 7.2 and earlier: HF_TOP=$EGO_TOP/eservice/hostfactory
+    HF_TOP=$EGO_TOP/hostfactory
+fi
+
+HF_VERSION=$( jetpack config symphony.hostfactory.version )
+if [ -z "${HF_VERSION}" ]; then
+    HF_VERSION="1.1"
+fi
+
+PKG_NAME=$( jetpack config symphony.pkg_plugin )
+if [ -f $PKG_NAME ]; then
+    rm -f $PKG_NAME
+    rm -rf packages
+    rm -rf hostfactory
+fi
+
+jetpack download --project symphony symphony/$PKG_NAME | exit 1;
+unzip $PKG_NAME
+pluginSrcPath=$HF_TOP/$HF_VERSION/providerplugins/azurecc
+VENV=$pluginSrcPath/venv
+export PATH=$(echo $PATH | sed -e 's/\/opt\/cycle\/jetpack\/system\/embedded\/bin://g' | sed -e 's/:\/opt\/cycle\/jetpack\/system\/embedded\/bin//g')
+export PATH=$PATH:/root/bin:/usr/bin
+if [[ "$(cat /etc/os-release)" == *"Ubuntu"* ]]; then
```

Comment:
Can remove since you added the python install script

Reply:
for some reason in ubuntu I had to install it here again while testing, but I can remove it if it works from python install script